x/auth/tx/service.go

```diff
@@ -135,6 +135,10 @@ func (s txServer) GetTx(ctx context.Context, req *txtypes.GetTxRequest) (*txtype
 	// https://github.com/cosmos/cosmos-sdk/issues/7036.
 	result, err := QueryTx(s.clientCtx, req.Hash)
 	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
```

Comment:
I checked and I don't think TM returns a typed error here.

Reply:
want to open an issue in tendermint for this? or you may have already?

Reply:
If yes, let's link that issue in an inline comment here